subsys/bluetooth/controller/include/ctrl_spec_def.h

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2020 Oticon A/S
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* Common specification definitions to be shared in controller */
+
+#ifndef	__CTRL_SPEC_DEF_H__
+#define	__CTRL_SPEC_DEF_H__
+
+#define	CTRL_SPEC_SLOT_TIME	625U		/* Magic number 625us */
+#define	CTRL_SPEC_CYCLE_TIME	1250U		/* Magic number 1250us */
```

Comment:
lll_internal.h appears to be vendor specific for only Nordic or Openisia. Since these are to be used controller wide (lll & ull), can they be placed in ll.h such that they are available controller wide?

Reply:
@cvinayak with regard to the previous comment, I would also like to complete this merge before the Christmas break next week if possible.

Could we agree on an organization of the definitions such that it can cover all the usage across the controller?

Options:
1) Centralized definition of the interval (new / existing header) only
    -- Where should be central definition be put such that it makes sense across the controller?

2) Centralized definition of the interval (new / existing header), redefined context specific at different layers / components.
    e.g.: https://github.com/zephyrproject-rtos/zephyr/pull/30639#discussion_r542058763
     -- Where should be central definition be put such that it makes sense across the controller?

3) Only context specific definitions (ull, lll, lll vendor specific / conn, scan, adv etc.) to be defined in the most relevant context specific header, no centralized control over the value

Thanks.

Reply:
@cvinayak a reminder on this.

Thanks.

Reply:
> lll_internal.h appears to be vendor specific for only Nordic or Openisia. Since these are to be used controller wide (lll & ull), can they be placed in ll.h such that they are available controller wide?

ll.h is exports to  files external to ll_sw folder, example used by HCI layer (hci.c, hci_driver.c)

Current convention to be followed is:
- If definitions are used inside LLL but specific to a role, define them in vendor specific h files inside `ll_sw/<vendor>/lll/lll_<role>_internal.h`.
- If definitions are used inside LLL but across roles, define them in vendor specific h files inside `ll_sw/<vendor>/lll/lll_internal.h`.
- If definitions are used inside LLL and ULL and/or Thread context (HCI driver) but specific to a role, define them in vendor specific h files inside `ll_sw/<vendor>/lll/lll_<role>.h`. Exception, if all of the definitions are common to vendors, the file can be `ll_sw/lll_<role>.h`
- If definitions are used inside LLL and ULL and/or Thread context (HCI driver) but across roles, define them in `ll_sw/lll.h` or `ll_sw/pdu.h`
- If definitions are used inside ULL but specific to a role, define them in `ll_sw/ull_<role>_internal.h`.
- If definitions are used inside ULL but across role, define them in `ll_sw/ull_internal.h`.
- If type definitions are used inside ULL and Thread context (HCI driver) but specific to a role, define them in `ll_sw/ull_<role>_types.h`.
- If definitions are used inside ULL and Thread context (HCI driver) but across roles, define them in `include/ll.h`

> Could we agree on an organization of the definitions such that it can cover all the usage across the controller?

Hope the above convention helps you define the macros in correct header files.